### ISSUE TITLE
Pull content and personal data into A-Frame and HTML for Scenario 1.

### ISSUE THREAD

**Issue by jasonalderman-valtech** [valtech-sd/spyglass#63]
Blocked by #33 and #34.

AC: 
- [x] Pull data from Contentstack for 
  - [x] Three serum product images
  - [x] Three serum product markers / .patt
  - [x] Serum testimonials
  - [x] _Serum ingredients / contraindications (stretch goal)_
- [x] Pull data from personalized data for 
  - [x] User's skincare goals
  - [x] Reasons why for each serum product
  - [x] Other selected products in skincare regimen
